### PR TITLE
Rework validator vote account defaults to half voting fees

### PR DESCRIPTION
**Pull request by mvines**
The vote account default setup encourages a validator to spend 100% more lamports in transaction fees than it really needs to.   Rework the defaults so that a vote requires 1 signature instead of 2.

Note that I **intentionally** break the `solana-validator` command-line arguments here, as I want to alert our validators of this change.  Plus the original command-line arguments felt weird.

Changes:
* `solana-genesis` now authorizes the bootstrap validator's identity account to vote, instead of the bootstrap validator's vote account itself.
* `solana create-vote-account`  now authorizes the validator's identity account to vote, instead of the new vote account itself.
* `solana-validator` argument changes:
        1. `--identity-keypair` is now `--identity`
        2. `--voting-keypair` is now `--authorized-voter` (matching the cli), now optional, defaulting to `--identity` 
        3. `--vote-account` is now required to vote (and no longer defaults to `--voting-keypair`)

tl;dr - here's what this means to your validator: https://github.com/solana-labs/cluster/pull/18/files

### CODE REVIEW

File: multinode-demo/bootstrap-validator.sh [hunk 1]
```diff
@@ -81,8 +81,9 @@ args+=(
   --ledger "$ledger_dir"
   --rpc-port 8899
   --snapshot-interval-slots 100
-  --identity-keypair "$identity_keypair"
-  --voting-keypair "$vote_keypair"
+  --identity "$identity"
+  --vote-account "$vote_account"
+  --authorized-voter "$vote_account"
```
Comment (t-nelson) [inline multinode-demo/bootstrap-validator.sh]: Should be `$identity` here?

Comment (mvines) [reply multinode-demo/bootstrap-validator.sh]: ah, thanks.  That was from my negative testing

**Comment by garious** [comment]
@aeyakovenko, drawing your attention to this change in particular:
> solana create-vote-account now authorizes the validator's identity account to vote, instead of the new vote account itself.

I wonder if that's okay. In discussions ages ago, we said the validator identity was supposed to be a throwaway key.  Not sure if that's changed.

**Comment by mvines** [comment]
> I wonder if that's okay. In discussions ages ago, we said the validator identity was supposed to be a throwaway key. Not sure if that's changed.

It's very much not throwaway.   It's **the** most important key you can have as a validator

**Comment by aeyakovenko** [comment]
we shouldn’t need two hot keys on every signature

**Comment by garious** [comment]
The most important keys are the ones we sent reward tokens to and anything authorized to vote (if slashing)

**Comment by mvines** [comment]
> The most important keys are the ones we sent reward tokens to and anything authorized to vote (if slashing)

which is the validator identity key :)

**Comment by garious** [comment]
Half the signatures to verify. I wonder if we'll see a drop in confirmation times.

**Comment by mvines** [comment]
Maybe my Raspberry PI validator can keep up with an idling cluster now
